webdev1/webdev3a.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+<!--    Update Map Title-->
+	<title>Leaflet Web Dev Maps</title>
+    
+<!--   Add course name, your name, etc -->
+    <h1>GIS 4533, WebDev Maps
+    <br>Brandy Meidl</br>Due 10/30/19</br>
+    </h1>
+    <!-- External Stylesheets -->
+    <!--Updated Leaflet version here-->
+    <link rel="stylesheet" href="https://unpkg.com/leaflet@1.5.1/dist/leaflet.css"
+  integrity="sha512-xwE/Az9zrjBIphAcBb3F6JVqxf46+CDLwfLMHloNu6KEQCAWi6HcDUbeOfBIptF7tcCzusKFjFw2yuvEpDL9wQ=="
+  crossorigin=""/>
+    
+    <!--Updated map sizing style here. Additional code commands noted below-->
+    <style>
+        body {
+            padding: 0;
+            margin: 0;
+        }
+        html, body, #map {
+            height: 100%;
+            width: 100%;
+        }
+    </style>
+    </head>
+<body>
+    
+    <!-- Our web map and content will go here -->
+    
+    <!--    Updated 2nd place of code for Leaflet version-->
+    <script src="https://unpkg.com/leaflet@1.5.1/dist/leaflet.js"
+  integrity="sha512-GffPMF3RvMeYyc1LWMHtK8EbPv0iNZ8/oTtHPx9/cc2ILxQ+u905qIwdpULaqDkyBKgOaB57QTMg7ztg8Jm2Og=="
+  crossorigin=""></script>
+    
+    <!-- Add the Leaflet JavaScript library -->
+    <a href="https://leafletjs.com">I built this with Leaflet</a>
+    <div id="map"></div>
+    
+    
+    
+    <script>
+        
+    // Create variable to hold map element, give initial settings to map
+    // Update map to Duluth coordinates
+    var map = L.map('map').setView(center: [42.362432, -71.086086], zoom: 14);
```

Comment:
Missing `{}` around options for `.setView`
I think this is what's breaking the map 🤕